README.md

```diff
@@ -72,8 +79,9 @@ Check the doc on [storageclasses](docs/storageclasses.md) to know all the suppor
 
 ##### Device Availability
 
```

Comment:
Can we have seperate heading explaing what is meta partition. Please include the following details there
1) It should have the portion number as one
2) It should not be formatted
3) Partition name cannot consist flags